### ISSUE TITLE
Custom key mapping `<c-.>` is not working

### ISSUE THREAD

**Issue by pingsoli** [philc/vimium#3835]
**Describe the bug**
I mapped `<c-.>` to `moveTabRight`, but it's not working.

**To Reproduce**
here is my code:
```
map <c-,> moveTabLeft count=99
map <c-.> moveTabRight count=99
```
<c-,> works like a charm, but <c-.> failed.

**Browser and Vimium version**
Chrome Vimium Version: 1.66

**Comment by gdh1995** [comment]
Maybe Ctrl+Period was blocked by Windows system and used to switch some modes of system IME.

**Comment by pingsoli** [comment]
That's right, it looks like `Ctrl + Period` is occupied by Windows system. I have no clue to fix it.
